test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageArchiveReaderTests.cs

```diff
@@ -1714,16 +1714,71 @@ public async Task ValidatePackageEntriesAsync_ValidPackageFiles_Succeeds()
         }
 
         [Fact]
-        public void GetContentHashForSignedPackage_WhenPackageNotSigned_ReturnsNull()
+        public void GetContentHash_WhenPackageNotSigned_ReturnsHashOfWholePackage()
```

Comment:
Do we have a test anywhere (out of context for this, I know) that assers the following 2 are equivalent. 
Given PackageA, when generating the hash, it returns unsighedHash. 
Given Signed PackageA, when generate the hash it returns a signedHash.